logger/logger.go

```diff
@@ -31,8 +31,10 @@ type Payload struct {
 
 // Line contains properties related to an individual log message.
 type Line struct {
+	Body      string `json:"line"`
```

Comment:
@dm36, can you make sure to use tabs/spaces to make this look well-aligned across the fields and values?

Reply:
we should be relying on `gofmt` to deal with this, it is the defacto standard for how go files should be formatted. this would fail the build via `golint` if it was incorrect

Reply:
changes here are fine - the reason this is formatted differently now is that `gofmt` formats "groups" of properties if they are separated by a newline. I don't have strong feelings on whether or not we separate with a newline, but that's the reason why both versions are correct 😄